src/Controls/src/Core/FormattedString.cs

```diff
@@ -41,9 +41,9 @@ void OnCollectionChanged(object sender, NotifyCollectionChangedEventArgs e)
 				foreach (object item in e.OldItems)
 				{
 					var bo = item as Span;
-					bo.Parent = null;
 					if (bo != null)
 					{
+						bo.Parent = null;
```

Comment:
Maybe also fix line 59? (That's why I personally prefer the `if (a is X x)` style instead of `as`.)